.gitignore

```diff
@@ -54,4 +54,7 @@ Temporary Items
 
 ### IntelliJ
 
-.idea
\ No newline at end of file
+.idea
+/node_modules/
+/dist/bundle.js
+/dist/index.html
```

Comment:
Mein Vorschlag wäre, den `dist` Ordner komplett in die .gitignore aufzunehmen. Die neue Datei `dist/filter.html` wird ja ebenfalls von webpack gneriert und sollte daher nicht mit eingecheckt werden.

Reply:
alles klar, done